es/src/main/scala/monix/connect/es/Elasticsearch.scala

```diff
@@ -0,0 +1,26 @@
+package monix.connect.es
+
+import com.sksamuel.elastic4s.bulk.BulkCompatibleRequest
+import com.sksamuel.elastic4s.http._
+import com.sksamuel.elastic4s.http.bulk.BulkResponse
+import monix.eval.Task
+
+/**
+  * Object for managing the Elasticsearch
+  */
+object Elasticsearch {
+  import com.sksamuel.elastic4s.http.ElasticDsl._
+
+  /**
+    * Execute bulk requests
+    *
+    * @param requests some [[BulkCompatibleRequest]]
+    * @param client an implicit instance of a [[ElasticClient]]
+    * @return a [[Task]] with [[Response]] with [[BulkResponse]]
+    */
+  def bulkRequest(requests: Seq[BulkCompatibleRequest])(
```

Comment:
what do you think abut adding more methods that will serve other type of single requests? 
(count, singleUpdate, refresh, search, merge, delete...)

Reply:
maybe we could also have an `ElasticSearchSource` or `ElasticSearchObservable` that will return _responses_ in form of `Observable`, not sure how to do so. 
It it seems like elastic4s provides a reactive api but that is backed with `Akka` and therefore we would need to create an `ActorSystem`, probably too heavy...

Reply:
OK. I'll add them.